packages/expo-in-app-purchases/src/InAppPurchases.ts

```diff
@@ -155,19 +157,25 @@ export async function getPurchaseHistoryAsync(
  * service on iOS.
  *
  * @param itemId The product ID of the item you want to buy.
- * @param oldPurchaseToken __Android Only.__ The `purchaseToken` of the purchase that the user is
- * upgrading or downgrading from. This is mandatory for replacing an old subscription such as when
- * a user upgrades from a monthly subscription to a yearly one that provides the same content.
- * You can get the purchase token from [`getPurchaseHistoryAsync`](#inapppurchasesgetpurchasehistoryasyncrefresh-boolean).
+ * @param details __Android Only.__ details for billing flow, can be one of:
```

Comment:
This wording is a little confusing and sounds like you can provide only one of the following options, when it looks like any combination of them is valid.

Reply:
Improved :)